README.md

```diff
@@ -17,12 +17,14 @@ This document describes how to build the FreeDV GUI program for various operatin
   $ cd freedv-gui
   $ ./build_linux.sh
   ```
+  (For Ubuntu 20.04 the wxWidgets package is named `libwxgtk3.0-gtk3-dev`.)
+  
```

Comment:
@jimt - have you actually built and run freedv-gui on Ubuntu 20?  I haven't tried this myself.

Reply:
As a side comment, I am now running well on Ubuntu 20. Mostly had to add in a bunch of Python2 and 3 stuff.

Reply:
Since moving to Ubuntu 20.04 I've only used FreeDV-GUI in test mode.

I haven't encountered any problems with Python, but I don't use `send_email_on_sync.py` (which appears to be Python 3 aware).

Reply:
Thanks @srsampson and @jimt.

@jimt - what do you mean by "test mode'?  I'm sure I should know what that is ....

Reply:
[Playing the supplied sample files](https://github.com/drowe67/freedv-gui/blob/master/USER_MANUAL.md#test-wave-files)  (in receive-only mode) to check the decode path without bringing the radio into the equation.

And it was then that I noticed there was a mode missing from this repo.  #66  :sunglasses:

Reply:
Yep that's the basic sanity check I do for freedv-gui too :+1: 

One day I'd like to add command line options for automated tests using freedv-gui, e.g. to decode a file without needed any mouse clicks.

Reply:
I've also had requests for PDF version of the manual.  There are tools for markdown->PDF however we'd need a way to automate the conversion every time we change the manual.  That's the nice thing about Markdown - always up to date!

Reply:
Needs some tweaking but how does this look?
https://hobbes1069.fedorapeople.org/USER_MANUAL.pdf

I generated it with pandoc, shouldn't be too hard to integrate it with the cmake config and generate it during building.

Reply:
For posterity here's the command line I used:
`$ pandoc USER_MANUAL.md -o USER_MANUAL.pdf "-fmarkdown-implicit_figures -o" --from=markdown -V geometry:margin=.4in --toc --highlight-style=espresso`

Reply:
Be worth putting that in the document.  It would be nice if we could have Travis rebuild a PDF version every time we push

Reply:
Although the file isn't "large", I would hate too keep checking in a 400-500k file into git every time the markdown is changed. Perhaps a simple bash script for convenience of the end user would be sufficient?

Reply:
Yep that's where I'm stuck too - don't want to manually maintain this step.  Many end users are Windows users who can't drive a Linux command line.  Some automation would be nice, e.g. rebuild PDF every time markdown changes (maybe bumping version number and release date or at least including hit hash) and pop it on the freedv.org web site.

Reply:
Oh well as a first step I added your command line to the manual in https://github.com/drowe67/freedv-gui/commit/82e0078059234d6e8cf462c21d9d0a5750e7dc02  @hobbes1069 

I figure someone in a local group of Windows FreeDV users will have the Linux skills could do the conversion for their friends.